script/Ciel/Day 9/09_01_CIEL09_4.txt

```diff
@@ -454,18 +456,18 @@ You're also an Executor, Sensei, so aren't you the same as Senpai?"
 [sha:7b06e52ee1f1c92025983bdc761c367705c5228e]{
 -- Page 412, Offset 23194.
 -- 「わたしが？　冗談やめて、あんなのと一緒にしないで。
-"Me? Please don't joke around, don't lump me with her.
+"Me? Please stop messing around. Don't lump me in with her.
 }
 [sha:783b03a0aed06690b286526549696e8d5510c901]{
 -- Page 412, Offset 23195.
 -- わたしはいたって普通の人間よ。あんな風にまわりを騙せないし、割り切るコトだってできない」
 // 割り切る make a clean distinction (between her Executor life and her pretend school girl life) without being influenced by personal emotions
-I'm a very normal human. I don't go around deceiving those around me like she does and I can't make such a cold calculus."
+I'm as normal as it gets. I can't just deceive the people around me as she does, and I can't even make a clear distinction either."
```

Comment:
maybe something more like
I can't just deceive the people around me or keep my work and personal life so separate.